### PR TITLE
Bump `got`.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
## Purpose of this pull request? 

- [ ] Documentation update
- [ ] Bug fix 
- [ ] Enhancement
- [x] Other, please explain: Bump dependency.

## What changes did you make?
Bump `got` from `^5.0.0` to `^6.7.1`.

## Is there anything you'd like reviewers to focus on?
I would like to make sure that the way I switched from a callback to a promise is proper.

### CODE REVIEW

File: lib/routes/install.js [hunk 1]
```diff
@@ -50,8 +50,8 @@ var getAllGenerators = _.memoize(function () {
 });
 
 function searchMatchingGenerators(app, term, cb) {
-  got('yeoman.io/blacklist.json', {json: true}, function (err, data) {
-    var blacklist = err ? [] : data;
+  function gotCallback(blacklist) {
+    console.log(blacklist);
```
Comment (SBoudrias) [inline lib/routes/install.js]: Left a debugging line here.

File: lib/routes/install.js [hunk 2]
```diff
@@ -50,8 +50,8 @@ var getAllGenerators = _.memoize(function () {
 });
 
 function searchMatchingGenerators(app, term, cb) {
-  got('yeoman.io/blacklist.json', {json: true}, function (err, data) {
-    var blacklist = err ? [] : data;
+  function gotCallback(blacklist) {
```
Comment (SBoudrias) [inline lib/routes/install.js]: Let's give this a better name... Maybe like `handleBlacklist`?

File: lib/routes/install.js (hunk 1 above)
Comment (wtgtybhertgeghgtwtg) [inline lib/routes/install.js]: I swear, I always miss these.

File: lib/routes/install.js [hunk 3]
```diff
@@ -67,7 +66,10 @@ function searchMatchingGenerators(app, term, cb) {
         return generatorMatchTerm(generator, term);
       }));
     }, cb);
-  });
+  }
+  got('yeoman.io/blacklist.json', {json: true})
```
Comment (SBoudrias) [inline lib/routes/install.js]: I wonder what's best practice now with got; `{json: true}` or doing it later with `response.json()` (just like browser `fetch()` API)

Comment (wtgtybhertgeghgtwtg) [inline lib/routes/install.js]: You'd have to ask sindresorhus.

**Comment by SBoudrias** [comment]
Thanks!

**Pull request merged**